core/src/epicli/cli/epicli.py

```diff
@@ -264,6 +270,22 @@ def dump_config(config):
         if attr.startswith('_'):
             logger.info('%s = %r' % (attr[1:], getattr(config, attr)))
 
+def ensure_vault_password_is_set(args):
+    vault_password = args.vault_password 
+    if vault_password is None:
+        vault_password = prompt_for_value("Provide password to encrypt vault: ")
```

Comment:
It would be better to use something for secure vault password reading without exposing it.